person.rb

```diff
@@ -1,8 +1,10 @@
-class Person
+require_relative 'nameable'
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] Console app is not running correctly due to the wrong number of parameters specified.  We want the Person class to take 2 arguments, _age_ and _name_ when instantiated. Apparently, your Person class can take either 1 or 3 arguments. Let's improve upon this aspect and ensure app is running correctly.